packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```
@@ -42,6 +42,10 @@ Parameters:
     Default: 'true'
     Type: 'String'
     AllowedValues: ['true', 'false']
+  InputPermissionsBoundary:
```

```suggestion
  PermissionsBoundary:
```

I think `Input` is implied